lnwallet/fee_estimator.go

```diff
@@ -180,6 +180,21 @@ func (b *BtcdFeeEstimator) fetchEstimatePerVSize(
 		return 0, err
 	}
 
+	// Before proceeding, we'll make sure that this fee rate respects the
+	// minimum relay fee set on the backend node.
+	info, err := b.btcdConn.GetInfo()
+	if err != nil {
+		return 0, err
+	}
+
+	// If the estimated fee rate is lower than the minimum relay fee rate,
+	// then we'll use that.
+	if minFeeRate, err := btcutil.NewAmount(info.RelayFee); err == nil {
```

Comment:
When would this fail?

Reply:
It *shouldn't*, but it doesn't hurt to check the error.